scripts/run.sh

```diff
@@ -64,6 +64,16 @@ check_files() {
     fi
 }
 
+# create the reports and logs folder
+create_folders() {
+    for dir in reports logs; do
```

Comment:
`logs` is not being used

```suggestion
    for dir in reports; do
```